Backpack/src/main/java/net/skyscanner/backpack/flare/BpkFlare.kt

```diff
@@ -144,8 +145,9 @@ open class BpkFlare @JvmOverloads constructor(
   }
 
   private fun initialize(attrs: AttributeSet?, defStyleAttr: Int) {
+    context.withBpkThemeInEditMode(isInEditMode)
```

Comment:
Can we move the call to constructor? Context used here and context passed to constructor might be different instances (for instance, if the parent widgets wraps the context), so I'd prefer to have a pattern.